app/main/views/direct_award.py

```diff
@@ -20,6 +20,11 @@ def list_projects():
     page = get_valid_page_or_1()
 
     projects = DirectAwardProject.query
+
+    with_users = convert_to_boolean(request.args.get('with-users', False))
```

Comment:
Can we use an `include=users` style argument instead, per <http://jsonapi.org/format/#fetching-includes>?

Reply:
Nice! I've not seen that before.